pkg/sources/sources.go

```diff
@@ -57,8 +57,10 @@ type GCSConfig struct {
 	ProjectID,
 	// ServiceAccount is the service account to use to authenticate with the source.
 	ServiceAccount string
+	// MaxObjectSize is the maximum object size to scan.
+	MaxObjectSize,
 	// Concurrency is the number of concurrent workers to use to scan the source.
-	Concurrency int
+	Concurrency int64
```

Comment:
Any reason for changing this?

Reply:
lol, i didn't want an int and int64 🤣 
switched it back.